appinventor/appengine/src/com/google/appinventor/client/actions/AboutAction.java

```diff
@@ -0,0 +1,60 @@
+package com.google.appinventor.client.actions;
```

Comment:
I know this one is technically my fault but we need to bring over the copyright headers for the individual Action classes from TopToolbar.java